### PR TITLE
Impact dashboard rough draft

### PR DESCRIPTION
**Pull request by zkenny**
## Description of the change

Implements facilities form and locale information sections of impact dashboard:

image

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/design-system/HelpButtonWithTooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components";
+
+import Colors from "./Colors";
+import Tooltip from "./Tooltip";
+
+const HelpButton = styled.span`
```
Comment (jessex) [inline src/design-system/HelpButtonWithTooltip.tsx:6]: A style note: in the mocks, this is always a little bit separate from the text label that it relates to, but in the current implementation it follows very closely from the text. Maybe just some left padding would suffice, or having it right-align to its container?

File: src/design-system/HelpButtonWithTooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  text-align: center;
+  font-size: 9px;
+
+  padding: 2px;
+`;
+
+const HelpButtonWithTooltip: React.FC = (props) => {
+  return (
+    <Tooltip content={props.children}>
+      <HelpButton>?</HelpButton>;
```
Comment (jessex) [inline src/design-system/HelpButtonWithTooltip.tsx:22]: This is putting an accidental semi-colon after every tooltip.

File: src/design-system/Input.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+export function useInputValue<T>(props: InputValueProps<T>) {
+  // We display the default value when valueEntered has never existed. A naive
+  // implementation would display the default value as long as valueEntered
+  // currently does not exist, but this means that if the user clears the entire
+  // input, it would immediately be populated with the default value -- this can
+  // be unexpected behavior.
+  let [shouldShowDefault, setShouldShowDefault] = useState(
+    props.valueEntered == null,
```
[the remark this answers is not in the document]
Comment (szhu) [reply src/design-system/Input.tsx:23]: `== null` is an idiomatic check for both `undefined` and `null`. It lines up well with the rather new `??` and `?.` operators, which also check for both `undefined` and `null`.

File: src/impact-dashboard/ImpactDashboard.tsx [hunk 4]
```diff
@@ -1,20 +1,257 @@
...
 
+const FacilityInformation: React.FC = () => {
+  return (
+    <FacilityInformationDiv>
+      <Description>
+        This section collects basic information about facility staff and your
+        incarcerated population by age and medical vulnerability. If you don't
+        have your in-facility population available by age brackets, enter your
+        overall population count in "Age unknown".
+      </Description>
```
[the remark this answers is not in the document]
Comment (szhu) [reply src/impact-dashboard/ImpactDashboard.tsx:185]: Padding is not finalized at all yet, thanks for looking at it though!

File: src/design-system/HelpButtonWithTooltip.tsx (hunk 1 above)
Comment (szhu) [reply src/design-system/HelpButtonWithTooltip.tsx:6]: Padding is not finalized at all yet

File: src/design-system/HelpButtonWithTooltip.tsx (hunk 2 above)
Comment (szhu) [reply src/design-system/HelpButtonWithTooltip.tsx:22]: wow, it was so tiny that I think I just kept on thinking there was dust on my screen. whoops

File: src/design-system/InputTextOld.tsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+interface Props {
+  type?: string;
+  label?: string;
+  placeholder?: string;
+  value?: string;
+  onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  defaultValue?: string;
+}
+
+const InputTextOld: React.FC<Props> = (props) => {
```
[the remark this answers is not in the document]
Comment (szhu) [reply src/design-system/InputTextOld.tsx:34]: I wanted to make a new props interface for `InputText`. Frances' form still used the old InputText props interface so I didn't delete it, I moved it to `InputTextOld`.

I'll add a TODO in this file to eventually remove this component.

File: src/impact-dashboard/ImpactDashboard.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (szhu) [reply src/impact-dashboard/ImpactDashboard.tsx:185]: I'll merge this PR now, but we'll track this in #46.

File: src/design-system/HelpButtonWithTooltip.tsx (hunk 1 above)
Comment (szhu) [reply src/design-system/HelpButtonWithTooltip.tsx:6]: I'll merge this PR now, but we'll track this in #46.

**Pull request merged**